Podfile.lock

```diff
@@ -1,36 +1,36 @@
 PODS:
-  - Firebase/CoreOnly (7.0.0):
-    - FirebaseCore (= 7.0.0)
-  - Firebase/Crashlytics (7.0.0):
+  - Firebase/CoreOnly (7.1.0):
+    - FirebaseCore (= 7.1.0)
+  - Firebase/Crashlytics (7.1.0):
```

Comment:
In the future, pod updates really should be committed as separate, atomic PRs. It makes this PR pretty far to review. As long as everyone only runs pod install locally, pods shouldn't be updated all the time. We could just run pod update once as an atomic PR prior to each release.

Reply:
It's part of this because it's still not released (this branch) and 7.0.0 had critical bug with crashes. :)